src/server/metrics.rs

```diff
@@ -12,6 +12,43 @@
 // limitations under the License.
 
 use prometheus::*;
+use prometheus_static_metric::*;
+
+make_static_metric! {
+    label_enum GrpcTypes {
```

Comment:
Maybe wait a bit longer, this feature is under review.

Reply:
Please take a look :)  It is really simple! Just argument forwardings.